Libraries/Components/TextInput/TextInput.js

```diff
@@ -333,6 +334,31 @@ type IOSProps = $ReadOnly<{|
   textContentType?: ?TextContentType,
 |}>;
 
+// [TODO(macOS GH#774)
+export type SubmitKeyEvent = $ReadOnly<{|
```

Comment:
It is not necessarily an event - but that's how it is called on Windows
https://github.com/microsoft/react-native-windows/pull/7333/files

CC: @rozele